10-information_governance.Rmd

```diff
@@ -0,0 +1,188 @@
+* __How many records does the data contain?__
+* __Who will have access to the data on the Analytical Platform?__
+* __How will the data be moved?__  
+  Describe how the data will be moved from the source location. Will the data be moved electronically, by email or by physical media? Will it be uploaded manually to S3 or a home directory or will there be a direct integration with another system?
+* __Is a technical migration form required?__  
+  A technical migration form may be required for recurring data movements or where there is an integration with other systems. If you are unsure whether your data movement requires a technical migration form, please contact the [Analytical Platform team](mailto:analytical-platform@digital.justice.gov.uk).
+* __Technical migration form__
+  Upload the complete technical migration form (maximum size 1GB). If you do not have an @digital email address and cannot upload a file to the form, please send the form as an attachment to the [Analytical Platform team](mailto:analytical-platform@digital.justice.gov.uk).
+* __Is this a one-off or recurring data movement?__
+* __When is the data movement expected to occur?__
+* __What is the intended frequency of the data movement?__
+* __When is the first data movement expected to occur?__
+* __Will the data movement be in place for a fixed period of time?__
+* __When will the final data movement occur?__
+* __When will the data movement request be reviewed?__  
+  Recurring data movements that occur indefinitely should be reviewed at least every two years.
+
+#### Data retention
+
+* __How long will the data be retained on the Analytical Platform?__
+* __Is this in line with departmental and Analytical Platform data retention policies?__
+
+#### Information Asset Owner (IAO)
+
+All data in MoJ is assigned an Information Asset Owner (IAO). The IAO is responsible for the registration, labelling, storage, transfer and retention of that data.
+You should seek approval from the relevant IAO for __all__ data movements.
+
+* __Does the data movement require approval by an Information Asset Owner (IAO)?__
+* __IAO name__
+* __IAO email address__
+* __Has the IAO approved this data movement?__
+
+#### Senior Information Risk Officer (SIRO)
+
+Information on the responsibilities of SIROs can be found [here](https://intranet.justice.gov.uk/guidance/knowledge-information/protecting-information/information-assurance-roles/).
+
+SIRO approval may be required depending on local information assurance policies. SIRO approval is usually required when a data movement involves non-routine risks. The relevant IAO should be able to advise if SIRO approval is also required.
+
+* __Does the data movement require approval by a Senior Information Risk Officer?__
+* __SIRO name__
+* __SIRO email address__
+* __Has the SIRO approved this data movement?__
+
+#### Information assurance
+
+* __Have you discussed the data movement with the relevant information security team(s)?__  
```

Comment:
Don't name people and put in their emails (given organisational change) - best to put in the team names and team emails (maybe link to People Finder for the team?)

Reply:
Does your team have a team mailbox I could link to? cybersecurity@digital.justice.gov.uk?

Reply:
cybersecurity@digital.justice.gov.uk is indeed the ticket

Reply:
Implemented in 4df0869.